numaprom/udf/window.py

```diff
@@ -12,7 +13,7 @@
 from numaprom.redis import get_redis_client
 from numaprom.tools import msg_forward, create_composite_keys, get_metric_config
 
-_LOGGER = logging.getLogger(__name__)
+LOGGER = logging.getLogger(__name__)
```

Comment:
any reason for this? Having an underscore as a prefix basically means that this variable is internal to the file itself and should not be imported

Reply:
aah okay, i just changed to make it same everywhere. Will make it _LOGGER on all the files.